src/libraries/System.Net.Quic/src/System/Net/Quic/Interop/msquic_generated.cs

```diff
@@ -2411,7 +2411,7 @@ internal byte AppCloseInProgress
                 }
 
                 [NativeTypeName("BOOLEAN : 1")]
-                internal byte ConnectionShutdownByPeer
+                internal byte ConnectionShutdownByApp
```

Comment:
I don't see this change in the original MsQuic sources nor the C equivalent.

Reply:
I'll remove it and return the `internal sealed class State` for now.

BTW, it's in this PR https://github.com/microsoft/msquic/pull/2883. I haven't had the chance to finish it (adding tests on Nick's request), but I do plan to publish it and propagate to dotnet/msquic and our docker images within 7.0 time frame.

Reply:
Done.